src/component/Beers.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+import { useState } from "react";
+import { useEffect } from "react";
+import { NavLink } from "react-router-dom";
+import Header from "./Header";
+
+function Beers() {
+
+const [beers, setBeers] = useState([]);
+
+useEffect(() => {
+    axios.get('https://ih-beers-api2.herokuapp.com/beers')
```

Comment:
For your project, it is great practice to have route saved in environment variables, like this

```
 axios.get(`${process.env.REACT_APP_API_BASE_URL}/beers`)
```

and accordingly in your `.env` file
```
REACT_APP_API_BASE_URL=https://ih-beers-api2.herokuapp.com
```